src/tools/sssctl/sssctl_cache.c

```diff
@@ -713,6 +758,49 @@ errno_t sssctl_netgroup_show(struct sss_cmdline *cmdline,
         return ret;
     }
 
+    return EOK;
+}
+
+errno_t sssctl_gpo_show(struct sss_cmdline *cmdline,
+                        struct sss_tool_ctx *tool_ctx,
+                        void *pvt)
+{
+    struct sssctl_cache_opts opts = {0};
+    const char *attr;
+    errno_t ret;
+
+    struct sssctl_object_info info[] = {
+        SSSCTL_CACHE_GPO_DESC,
+        SSSCTL_CACHE_GPO_GUID,
+        SSSCTL_CACHE_GPO_PATH,
+        SSSCTL_CACHE_GPO_VERSION,
+        SSSCTL_CACHE_GPO_TIMEOUT,
+        SSSCTL_CACHE_NULL
+    };
+
+    struct poptOption options[] = {
+        {"guid", 'g', POPT_ARG_NONE, &opts.guid, 0, _("Search by GPO guid"), NULL },
+        POPT_TABLEEND
+    };
+
+    ret = parse_cmdline(cmdline, tool_ctx, options, &opts.value, &opts.domain);
+    if (ret != EOK) {
+        ERROR("Failed to parse command line: %s", sss_strerror(ret));
```

Comment:
Hi,

please add `\n` to the end of `ERROR` messages (e.g. below as well) because they are not added automatically.

bye,
Sumit